dev-lang/swig/swig-4.0.2.ebuild

```diff
@@ -16,11 +16,11 @@ IUSE="ccache doc pcre"
 RESTRICT="test"
 
 RDEPEND="
-	virtual/pkgconfig
 	pcre? ( dev-libs/libpcre )
 	ccache? ( sys-libs/zlib )
 "
 DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
```

Comment:
Are you sure this is right? swig does runtime compilation stuff, RDEPEND pkgconfig could make sense

Reply:
Yeah, seems fine.